src/components/RadicleId.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
```

Comment:
We have a duplicate `RadicleId.svelte` now. The one in `/src/components/` should be the only one.